standard/statements.md

```diff
@@ -1388,6 +1397,8 @@ finally
 }
 ```
 
+In either expansion, an implementation may determine that a `CancellationToken` should be passed to the `GetAsyncEnumerator` method. This may include a combined token as shown in the example in §enumerator-cancellation. This enables the type implementing `IAsyncEnumerator` to implement cancellation as part of the implementation of `MoveNextAsync`.
```

Comment:
Note for @jskeet and @Nigel-Ecma 

This still feels a bit hand-wavy to me. Any suggestions are welcome.